CONTRIBUTING.rst

```diff
@@ -1,29 +1,24 @@
 Developing
 ==========
 
-Install dependencies:
+This project uses `Tox <https://tox.readthedocs.io/en/latest/config.html>`_ to manage virtual environments.
 
-::
+To run the tests::
 
-    pip install treq[dev]
+    tox -e py38-twisted_latest
 
-Run Tests (unit & integration):
+Lint::
 
-::
-
-    trial treq
-
-Lint:
-
-::
-
-    pep8 treq
-    pyflakes treq
+    tox -e flake8
 
 Build docs::
 
     tox -e docs
 
+To do it all::
+
+    tox -p
```

Comment:
On my tox version I need to explicitly specify `all` or a number of parallel builds 

```
$ tox --version
3.13.2
```

```suggestion
    tox -p all
```

Reply:
You should update your Tox. :) I don't need to pass anything on 3.20.1; it defaults to the number of CPU on my machine. `all` is semantically different: it runs all the jobs in parallel, regardless of available resources.

Reply:
you can set a minversion to avoid this: https://tox.readthedocs.io/en/latest/config.html#conf-minversion